app/src/test/java/com/kickstarter/libs/LakeTest.kt

```diff
@@ -475,6 +475,31 @@ class LakeTest : KSRobolectricTestCase() {
         this.lakeTest.assertValues("Project Page Pledge Button Clicked")
     }
 
+    @Test
+    fun testVideoProperties() {
```

Comment:
This is great @hadia is perfect! just a small thing, can you move this test from `LakeTest` to `SegmentTest` or duplicate it as you wish, Lake client will be deleted some time in the near future alongside with the tests for it

Reply:
sure